webpack/components/SnapshotManagement/components/SnapshotForm/SnapshotForm.js

```diff
@@ -50,9 +50,10 @@ const SnapshotForm = ({
       url: SNAPSHOT_CREATE_URL.replace(':hostId', hostId),
       values: submitValues,
       message: __('Snapshot successfully created!'),
+      successCallback: props.setModalClosed,
+      actions,
     });
     actions.setSubmitting(false);
-    props.setModalClosed();
```

Comment:
Moving this is not compatible with forman < 2.5, because `submitForm()` does not seem to support the successCallback.

We could keep the `setModalClosed()`, here and be compatible with foreman < 2.5

The nice thing with setting it as `successCallback` is the fact that the modal will stay open until the Snapshot has been created instead of closing immediately after hitting Submit.